packages/mobile/src/analytics/Events.tsx

```diff
@@ -98,8 +98,16 @@ export enum OnboardingEvents {
   pin_never_set = 'pin_never_set',
 
   wallet_import_start = 'wallet_import_start',
-  wallet_import_complete = 'wallet_import_complete',
-  wallet_import_cancel = 'wallet_import_cancel', // when a user cancels import of 0 balance wallet
+  wallet_import_phrase_updated = 'wallet_import_phrase_updated',
+  wallet_import_submit = 'wallet_import_submit',
+  wallet_import_cancel = 'wallet_import_cancel', // when a user cancels import of empty wallet or naviagtes back
```

Comment:
nit: There's a small typo: `naviagtes` -> `navigates`

Reply:
Resolved in 87f690d